test/parallel/test-http-many-ended-pipelines.js

```diff
@@ -15,10 +15,27 @@ var numRequests = 20;
 var done = 0;
 
 var server = http.createServer(function(req, res) {
-  res.end('ok');
+
+  var callbackCalled = false;
+
+  res.end('ok', function() {
+    assert.ok(!callbackCalled);
+    callbackCalled = true;
+  });
+
+  // We might get a socket already closed error here
+  // Not really a surpise
+  res.on('error', function() {});
```

Comment:
Failing to write some data will now always create an error. Which is a change that would've broken the test.

Previously there would've been no error even though the data was never written.